tasties_app/tests/conftest.py

```diff
@@ -183,3 +183,23 @@ def recipe():
 def signed_up_credentials():
     user = User.objects.create_user(username=VALID_USER, email=VALID_EMAIL, password=VALID_PASSWORD)
     user.save()
+
+
+@pytest.fixture
+def salad_recipes_with_rating():
+    return Recipe.objects.filter(title__icontains='salad')
+
+
+@pytest.fixture
+def empty_search():
+    return len(Recipe.objects.filter(title__icontains='abcdefg'))
+
+
+@pytest.fixture
```

Comment:
The fixture need to be only for create a data in the database.
All the tast need to be in anther file and anther funtction in the new file you will create.
Please take a look on work we did befor and take inspiration from there.